posts/2023/office-hours-help.md

```diff
@@ -0,0 +1,75 @@
+
+Select "Add New Calendar" which will open a window to add your name, email, description of the kind of support you can provide, and import a photo (please make it square for a consistent look). Save your calendar. You can re-navigate to this window by selecting "Calendar Settings" at any time.
+![](../../images/office-hours-help/calendarsettings.png)
+
+Then click on the right-facing arrow on your calendar to expand it to show your hours and availability settings.
+![](../../images/office-hours-help/setavailabilitylimits.png)
+
+You'll see that your "Appointment Types" should be set to "Earth System Data Science (ESDS) Office Hours" by default, if not you can fix this by selecting "Edit Group."
+
+Next, select "Edit Availability/Limits".
+
+This will bring you to a window titled "Set Hours of Availability." Here you should edit and save your regular hours. These can be whatever you like, generally a subset of 9-5 M-F. You can choose to block of lunches, first thing in the morning, last thing on Friday before the weekend up to you.
+![](../../images/office-hours-help/sethours.png)
+
+If you scroll down here you can set hours for specific days, to block of a week or some time you don't feel you have the capacity to help out with an Office Hours appointment. Vacations and other conflicts are inherited from your Google calendar (we'll cover that soon).
+
+Optionally, you can toggle over to "Calendar Scheduling Limits." This gives you options to set your max number of appointments per week or day so that you aren't ever giving up more than X numbers a week to your core work.
+![](../../images/office-hours-help/limits.png)
+
+Next we need to set up the two-way sync with your UCAR Google calendar. This is found on the left nav bar in "Business Settings" --> "Sync with other Calendars".
+![](../../images/office-hours-help/sync.png)
+
+Find your appointment calendar, and select a calendar to sync with. Only you have access to sync to your personal calendars, so this step cannot be done for you.
+![](../../images/office-hours-help/selectcalendar.png)
+
+Then, it is important to scroll down and check "Hide event title and only show "Busy" to other users" so that the rest of the ESDS Office Hours doesn't have the ability to see your calendar details (just that you are busy, but not with what). Then click "Save."
+![](../../images/office-hours-help/hidetitles.png)
+
+And that's it! You can double check that it worked by navigating to the [ESDS Office Hours scheduling page](office-hours/) to see your image there.
+
+## Want to Create a New Appointment Type?
+
+If you are another team at NCAR, you can share our Acuity Squarespace scheduling account. Our account supports up to 36 different calendars (office hours assistants), so as long as our combined usage is less than that we can save $$ by sharing.
+
+In fact we already share with the HPC Research Computing Support group. We can do this by adding additional appointment types to the account.
```

Comment:
Maybe add a link to the HPC "office hours" here? https://arc.ucar.edu/knowledge_base/121077762